repair/row_level.cc

```diff
@@ -521,7 +521,15 @@ class repair_writer {
                 return consumer(std::move(reader));
             });
         },
-        t.stream_in_progress());
+        t.stream_in_progress()).then([this, node_idx] (uint64_t partitions) {
```

Comment:
I see that you do not like using then_wrapped() :) On a fast path it is important. I hope this one is not such.

Reply:
It is on a slow path. Right, I am not a fan of then_wrapped. I use it only when I have no choice.

Reply:
It saves compilation time and the code size as well...